app/build.gradle

```diff
@@ -185,8 +189,12 @@ dependencies {
 
     compile "android.arch.lifecycle:extensions:$arch_version"
 
-    compile "android.arch.persistence.room:runtime:$arch_version"
-    kapt "android.arch.persistence.room:compiler:$arch_version"
+    compile "android.arch.persistence.room:runtime:$room_version"
```

Comment:
room is on 1.0.0, arch on 1.1.0

Reply:
was afraid that this causes problems so I never updated arch - hoped for room 1.1.0-alpha to come out of alpha soon.

Reply:
In the demo they use arch_core and room as version labels: https://github.com/googlesamples/android-architecture-components/blob/master/versions.gradle#L25